### DISCUSSION TITLE
copilot not renewing

### DISCUSSION THREAD

**Discussion by matinSurchi** [community/community#47130] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have renewed my payment yet still copilot is not renewing in my vs code, what should I do

**Comment by jge162** [comment]
you need to reach out to [GitHub Support](https://support.github.com/contact/account?tags=hubberfy_account) Billing issues.

**Comment by davecheney** [comment, marked as answer]
@matinSurchi you card was declined.
